app/controllers/api/v1/users_controller.rb

```diff
@@ -120,4 +120,24 @@ def update
     standard_update(User, current_human_user.id)
   end
 
-end
\ No newline at end of file
+  ###############################################################
+  # find_unclaimed
+  ###############################################################
+
+  api :POST, '/unclaimed/:email', 'Finds or creates a user by email address.'
```

Comment:
And won't the calling app want an application user? (to fit into the rest of its logic?)